RunCat365/Program.cs

```diff
@@ -111,6 +115,83 @@ private static Theme GetSystemTheme()
             return (int)value == 0 ? Theme.Dark : Theme.Light;
         }
 
+        private static async Task<bool> GetStartupAsync() {
+            if (IsRunningAsPackaged()) {
+                if (startupTask is null) startupTask = await StartupTask.GetAsync("RunCatStartup");
+                if (startupTask is null) return false;
+                if (startupTask.State == StartupTaskState.Enabled) return true;
+                return false;
+            } else {
+                var keyName = @"Software\Microsoft\Windows\CurrentVersion\Run";
+                using var rKey = Registry.CurrentUser.OpenSubKey(keyName);
+                if (rKey is null) return false;
+                var value = (rKey.GetValue(Application.ProductName) is not null);
+                rKey.Close();
+                return value;
+            }
+        }
+
+        private static async Task<bool> SetStartupAsync(bool isChecked) {
+            if (IsRunningAsPackaged()) {
+                bool active = !isChecked;
+                bool changeCheck = false;
+                if (startupTask is null) startupTask = await StartupTask.GetAsync("RunCatStartup");
+                if (active) {
+                    switch (startupTask.State) {
+                        case StartupTaskState.Enabled:
+                            changeCheck = true;
+                            break;
+                        case StartupTaskState.Disabled:
+                            StartupTaskState newState = await startupTask.RequestEnableAsync();
+                            if (newState == StartupTaskState.Enabled) {
+                                changeCheck = true;
+                            } else {
+                                MessageBox.Show("Launch at Startup could not be activated.", "Warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                                changeCheck = false;
+                            }
+                            break;
+                        case StartupTaskState.DisabledByUser:
+                            MessageBox.Show("Launch at startup was disabled by the user, enable it in Task Manager > Startup, search RunCat365 and enable it.", "Warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                            changeCheck = false;
+                            break;
+                        case StartupTaskState.DisabledByPolicy:
+                            MessageBox.Show("Launch at startup was disabled by policy.", "Warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                            changeCheck = false;
+                            break;
+                    }
+                } else if (!active) {
```

Comment:
This can be simplified to just `else`, since `active` is a boolean.
```suggestion
                } else {
```